app/Models/ChatMessageModel.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Ultra_Saver;
+
+public class ChatMessageModel
+{
+    [Key]
+    public int Id { get; set; }
+    public string Email { get; set; } = null!;
+
+    public string User { get; set; } = null!;
+
+    public string Message { get; set; } = null!;
+
+    public string? Updated_at { get; set; }
```

Comment:
Breaks C#'s (and our) convention, should be UpdatedAt

Reply:
Will fix in #92